src/main/js/field/StatusMessage.js

```diff
@@ -17,8 +17,7 @@ class StatusMessage extends Component {
     componentDidUpdate(prevProps) {
         if (this.props.errorMessage !== prevProps.errorMessage && !this.state.showError) {
             this.setState({ showError: true });
-        }
-        if (this.props.actionMessage !== prevProps.actionMessage && !this.state.showMessage) {
+        } else if (this.props.actionMessage !== prevProps.actionMessage && !this.state.showMessage) {
```

Comment:
Show either the error message or the action message.  Error message takes precedence.